pkgs/games/factorio/default.nix

```diff
@@ -225,9 +227,19 @@ let
         wayland
       ];
 
+      optimizationArgs =  ''
+        --prefix LD_PRELOAD : ${mimalloc}/lib/libmimalloc.so       \
```

Comment:
Is this a trick that is commonly used in this game community?

Reply:
Not too much, as people don't know about it and others wouldn't want to launch their game via shell script.

Reply:
Not to mention doing this with the steam version while not impossible is a lot harder.